### PR TITLE
Fixes #33011 - Navigation doesn't display Host Wizard without admin rights

### PR DESCRIPTION
**Pull request by Andrewgdewar**
Steps to Reproduce:

1. Create a non-admin user
2. Login with said user
3. Attempt to view Lab Features > Host Wizard

### CODE REVIEW

**Comment by xprazak2** [comment]
[ok to test]

File: app/registries/menu/loader.rb [hunk 1]
```diff
@@ -98,6 +98,7 @@ def self.load
         menu.sub_menu :lab_features_menu, :caption => N_('Lab Features'), :icon => 'fa fa-flask' do
           menu.item :host_wizard,
             :caption => 'Host Wizard',
+            :url_hash => {:controller => 'api/v2/hosts', :action => 'create'},
```
Comment (ezr-ondrej) [inline app/registries/menu/loader.rb]: Can you add spaces pls? Idk why Rubocop is not complaining about it, but it should and probably would once it wakes up xD
```suggestion
            :url_hash => { :controller => 'api/v2/hosts', :action => 'create' },
```

**Review by ezr-ondrej — COMMENTED** [review]
The host wizard does nothing as of today, so it was not so crucial to permission guard it, but I agree it should be in place :)
So thanks for the addition @Andrewgdewar :+1:

File: app/registries/menu/loader.rb (hunk 1 above)
Comment (Andrewgdewar) [reply app/registries/menu/loader.rb]: Lol, will do! My ruby environment is likely still not setup correctly.

**Comment by ezr-ondrej** [comment]
ugh? :) was the close intentional @Andrewgdewar ?

**Comment by Andrewgdewar** [comment]
no... ugh

**Comment by Andrewgdewar** [comment]
Here's a new one... le sigh.
https://github.com/theforeman/foreman/pull/8666

**Pull request closed without merging**